addons/hr_contract_calendar/models/hr_employee.py

```diff
@@ -0,0 +1,40 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from pytz import timezone
+from datetime import datetime, time
+from dateutil.relativedelta import relativedelta
+
+from odoo import models
+
+
+class HrEmployeeBase(models.AbstractModel):
+    _inherit = "hr.employee.base"
+
+    def _get_calendar(self, start=None, stop=None):
+        self.ensure_one()
+        if not start:
+            return False
+        if not stop:
+            stop = start
+        return self.sudo()._get_contracts(
+            start.replace(hour=0, minute=0, second=0),
+            stop.replace(hour=23, minute=59, second=59),
+            states=['open', 'close']).resource_calendar_id
```

Comment:
hum hum... Imagine employee has contract that starts next week, that contract is not either running or closed, so, we won't that contract here.. and I am guessing, that's not what we want?

Reply:
as we said, I check the closed contract to show the history of employee's schedule. And a new contract is a contract in draft so I don't want to show its schedule in the calendar :D